app/HutchAgent/appsettings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "WfexsOptions": {
+    "ExecutorPath": "/home/vasiliki/WfExS-backend/",
```

Comment:
I don't think any of these should be present in the committed file.

- If you're overriding for your own development purposes, use User Secrets
- if they are defaults, set them as default in the Options Model
- if they are local development defaults, use an `appsettings.Development.json`